llms/mlx_lm/tokenizer_utils.py

```diff
@@ -193,7 +193,9 @@ def reset(self):
         self.tokens = []
 
     def _maybe_trim_space(self, current_text):
-        if current_text[0] != " ":
+        if len(current_text) < 1 or current_text is None:
```

Comment:
If the `current_text` is ever `None` then taking the `len` will raise an exception, so I don't think it makes sense to check for that.

Reply:
Thanks, makes sense and the if condition improvement you made looks good to me.